src/main/java/Storage.java

```diff
@@ -0,0 +1,90 @@
+package duke;
+
+import duke.Task;
+import duke.DukeException;
+import duke.TaskList;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+import java.util.ArrayList;
+import java.util.Optional;
+import java.util.Scanner;
+
+/**
+ * Storage deals with loading and saving task list from/ to memory.
+ */
+
+public class Storage {
+	private String filePath;
+
+	public Storage(String filePath) {
+		this.filePath = filePath;
+	}
+
+	public ArrayList<Task> load() throws DukeException {
+		File file = new File(filePath);
+		ArrayList<Task> list = new ArrayList<>();
```

Comment:
i think my tutor mentioned that 'list' as a collection name isnt very intuitive and informative.